data/campaigns/Eastern_Invasion/scenarios/07b_Undead_Crossing.cfg

```diff
@@ -131,7 +131,7 @@
 
         [message]
             speaker=Mal-un-Karad
-            message= _ "Creatures from the deep, I summon thee to destroy this foe!"
+            message= _ "Creatures from the deep, I summon you to destroy this foe!"
```

Comment:
"thee" does sound more lich-like to me

Reply:
I changed it because "thee" is a singular pronoun and "Creatures from the deep" is a plural noun phrase. Do liches not follow that grammatical rule?

Reply:
You're right, my mistake.

Reply:
"thee" sounds better, I think, even if it's slightly ungrammatical.

Reply:
Done, thanks.